docs/reference-guides/theme-json-reference/theme-json-living.md

```diff
@@ -39,14 +39,14 @@ Setting that enables the following UI tools:
 
 ---
 
-### useRootPaddingAwareAlignments
```

Comment:
`useRootPaddingAwareAlignments` is removed in this PR, but returned in #63639. I wanted to keep the codegen changes in a separate PR, but the existing codegen didn't support the refactored structure which is why it is missing here.